backend/datautil/sqlite_to_mysql.py

```diff
@@ -51,10 +60,36 @@ def export_to_mysql():
                               row["INSTRUCTOR"],
                               row["DESCRIPTION"],
                               ))
+    """
+    SQLITE CAPES_DATA TO MYSQL CAPES_DATA 
+    """
+
+    mysql_cursor.execute("DROP TABLE IF EXISTS CAPES_DATA")
+    mysql_cursor.execute("CREATE TABLE CAPES_DATA"
+                        "(DEPARTMENT VARCHAR(255), COURSE_NUM VARCHAR(255), INSTRUCTOR TEXT, "
```

Comment:
Consistency. You had VARCHAR for departments and course_num sections for the CLASS_DATA table, so I thought it would be appropriate to have those distinctions carry over.